src/i18n/index.ts

```diff
@@ -1,7 +1 @@
-import en from './en/translation.json';
-import fr from './fr/translation.json';
 
```

Comment:
Why did you remove the content of this file?

Reply:
I moved it to the lang.ts